tests/conftest.py

```diff
@@ -42,8 +42,10 @@ def _sample_svs(tmpdir_factory):
 
 @pytest.fixture(scope="session")
 def _sample_jp2(tmpdir_factory):
-    """Sample pytest fixture for JP2 images. Download ndpi image for pytest."""
+    """Sample pytest fixture for JP2 images. Download jp2 image for pytest."""
+
     jp2_file_path = tmpdir_factory.mktemp("data").join("test1.jp2")
+
     if not pathlib.Path(jp2_file_path).is_file():
         print("\nDownloading JP2")
         r = requests.get("https://tiatoolbox.dcs.warwick.ac.uk/sample_wsis/test2.jp2")
```

Comment:
Why not we rename this file as test1 as well? This is being saved as test1.jp2 in jp2_file_path